pages/favorites/index.js

```diff
@@ -4,25 +4,26 @@ import SpotCard from "../../components/SpotCard.js";
 export default function FavoriteSpots({ spots, toggleBookmark }) {
   return (
     <StyledBookmarkSection>
-      <ul>
+      <SpotCard>
```

Comment:
You are nesting `SpotCard` inside `SpotCard`, that's why it's not rendering the spot info!

So altogether:

```jsx
<StyledBookmarkSection>
  {spots
    .filter((spot) => spot.isBookmark)
    .map((spot) => (
      <SpotCard spot={spot} key={spot.id} toggleBookmark={toggleBookmark} />
    ))}
</StyledBookmarkSection>
```